javascriptv3/example_code/cross-services/ddb-item-tracker/public/items.html

```diff
@@ -0,0 +1,237 @@
+    </div>
+  </div>
+</nav>
+
+<div class="container" id="manageItems">
+
+    <h3>Welcome to AWS DynamoDB Item Tracker</h3>
+    <h3 id="info3">Get Items</h3>
+    <p>You can manage items in this view.</p>
+
+    <table id="myTable" class="display" style="width:100%">
+        <thead>
+        <tr>
+            <th>Item Id</th>
+            <th>Guide</th>
+            <th>Date Created</th>
+            <th>Description</th>
+            <th>Status</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>No Data</td>
+            <td>No Data </td>
+            <td>No Data</td>
+            <td>No Data</td>
+            <td>No Data</td>
+        </tr>
+        </tbody>
+        <tfoot>
+        <tr>
+            <th>Item Id</th>
+            <th>Guide</th>
+            <th>Date Created</th>
+            <th>Description</th>
+            <th>Status</th>
+        </tr>
+        </tfoot>
+        <div id="success3"></div>
+    </table>
+
+</div>
+<br>
+<div id="modform" class="container" >
+
+    <h3>Modify an Item</h3>
+    <p>You can modify items.</p>
+
+    <form>
+        <div class="control-group">
+            <div class="form-group floating-label-form-group controls mb-0 pb-2">
+                <label>ID</label>
+                <input class="form-control" id="id" type="id" placeholder="Id" readonly data-validation-required-message="Item Id.">
+                <p class="help-block text-danger"></p>
+            </div>
+        </div>
+        <div class="control-group">
+            <div class="form-group floating-label-form-group controls mb-0 pb-2">
+                <label>Description</label>
+                <textarea class="form-control" id="description" rows="5" placeholder="Description"  data-validation-required-message="Description."></textarea>
+                <p class="help-block text-danger"></p>
+            </div>
+        </div>
+        <div class="control-group">
+            <div class="form-group floating-label-form-group controls mb-0 pb-2">
+                <label>Status</label>
+                <textarea class="form-control" id="status" rows="5" placeholder="Status" required="required" data-validation-required-message="Status"></textarea>
+                <p class="help-block text-danger"></p>
+            </div>
+        </div>
+        <br>
+    </form>
+
+</div>
+
+<div id="dialogtemplate2" border="2" title="Basic dialog">
+
+    <table  align="center">
+        <tr>
+            <td>
+                <p>Options:</p>
+            </td>
+            <td>
+
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>Select Manager:</p>
+            </td>
+            <td>
+
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <select id="manager">
+                    <option value="brmur@amazon.com">brmur@amazon.com</option>
+                    <option value="susfer@amazon.com">swhite@nomail.com</option>
```

Comment:
Probably don't want personal email addresses in the example!

Reply:
done